### PR TITLE
[INFRA] Update ANNOUNCE mail template

### PR DESCRIPTION
**Pull request by pan3793**
### _Why are the changes needed?_

Update ANNOUNCE mail template to add more detailed descriptions of Kyuubi.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: build/release/tmpl/announce.tmpl [hunk 1]
```diff
@@ -20,11 +20,30 @@ Title: [ANNOUNCE] Release Apache Kyuubi(Incubating) {release_version}
...
 Apache Kyuubi(Incubating) is a distributed multi-tenant JDBC server for
-large-scale data processing and analytics, built on top of Apache Spark.
+large-scale data processing and analytics, built on top of Apache Spark
+and designed to support more engines (i.e. Flink).
+
+Kyuubi provides a pure SQL gateway through Thrift JDBC/ODBC interface
+for end-users to manipulate large-scale data with pre-programmed and
+extensible Spark SQL engines.
+
+We are aiming to make Kyuubi an “out-of-the-box” tool for data warehouses
```
Comment (timothy65535) [inline build/release/tmpl/announce.tmpl:38]: nit: `“` to `"`

File: build/release/tmpl/announce.tmpl [hunk 2]
```diff
@@ -20,11 +20,30 @@ Title: [ANNOUNCE] Release Apache Kyuubi(Incubating) {release_version}
...
+and data lakes.
+
+This “out-of-the-box” model minimizes the barriers and costs for end-users
+to use Spark at the client side.
+
+At the server-side, Kyuubi server and engines' multi-tenant architecture
```
Comment (timothy65535) [inline build/release/tmpl/announce.tmpl:44]: nit: `engines'` to `engine's`

File: build/release/tmpl/announce.tmpl [hunk 3]
```diff
@@ -20,11 +20,30 @@ Title: [ANNOUNCE] Release Apache Kyuubi(Incubating) {release_version}
...
+and designed to support more engines (i.e. Flink).
+
+Kyuubi provides a pure SQL gateway through Thrift JDBC/ODBC interface
+for end-users to manipulate large-scale data with pre-programmed and
+extensible Spark SQL engines.
+
+We are aiming to make Kyuubi an “out-of-the-box” tool for data warehouses
+and data lakes.
+
+This “out-of-the-box” model minimizes the barriers and costs for end-users
```
Comment (timothy65535) [inline build/release/tmpl/announce.tmpl:41]: ditto

File: build/release/tmpl/announce.tmpl [hunk 4]
```diff
@@ -20,11 +20,30 @@ Title: [ANNOUNCE] Release Apache Kyuubi(Incubating) {release_version}
 Content:
 Hi all,
 
+# (To user@spark.apache.org)
+# Hello Spark Community,
```
Comment (timothy65535) [inline build/release/tmpl/announce.tmpl:24]: `Spark Community` ?

Comment (pan3793) [reply build/release/tmpl/announce.tmpl:24]: Yea, we are planning to send this mail to `dev@kyuubi`, `general@incubator`, and `user@spark`.

Comment (timothy65535) [reply build/release/tmpl/announce.tmpl:24]: Got

**Review by ulysses-you — APPROVED** [review]

**Comment by ulysses-you** [comment]
thanks, merging to master

**Comment by pan3793** [comment]
Also port to 1.3

**Pull request closed without merging**